http://svn.gna.org/viewcvs/gnustep?view=revision&revision=32788
* Modules/Parsers/ProjectCenter/PCParser.m [-parse]: Move the
selector and IMP definitions inside the method. Global IMP caching
is always wrong! And this broke compilation with llvm.

http://svn.gna.org/viewcvs/gnustep?view=revision&revision=33818
use setAllowedFileTypes method instead of private method which was removed

--- apps/projectcenter/trunk/Modules/Parsers/ProjectCenter/PCParser.m	2011/04/06 18:40:59	32787
+++ apps/projectcenter/trunk/Modules/Parsers/ProjectCenter/PCParser.m	2011/04/06 18:46:03	32788
@@ -124,23 +124,21 @@
     } 
 }
 
-static SEL selString = @selector(string:);
-static SEL selNumber = @selector(number:);
-static SEL selSpaceAndNewLine = @selector(spaceAndNewLine:);
-static SEL selInvisible = @selector(invisible:);
-static SEL selSymbol = @selector(symbol:);
-
-static void (*impString)(id, SEL, id);
-static void (*impNumber)(id, SEL, id);
-static void (*impSpaceAndNewLine)(id, SEL, unichar); 
-static void (*impInvisible)(id, SEL, unichar); 
-static void (*impSymbol)(id, SEL, unichar); 
-
 - (void)parse
 {
   unsigned int i, start, end;
   CodeType startType, endType;
   NSString *out;
+  SEL selString = @selector(string:);
+  SEL selNumber = @selector(number:);
+  SEL selSpaceAndNewLine = @selector(spaceAndNewLine:);
+  SEL selInvisible = @selector(invisible:);
+  SEL selSymbol = @selector(symbol:);
+  void (*impString)(id, SEL, id);
+  void (*impNumber)(id, SEL, id);
+  void (*impSpaceAndNewLine)(id, SEL, unichar); 
+  void (*impInvisible)(id, SEL, unichar); 
+  void (*impSymbol)(id, SEL, unichar); 
 
   NSLog(@"CodeParser begin...");
 
--- apps/projectcenter/trunk/Framework/PCAddFilesPanel.m	2011/09/08 05:59:20	33817
+++ apps/projectcenter/trunk/Framework/PCAddFilesPanel.m	2011/09/08 13:05:29	33818
@@ -1,9 +1,10 @@
 /*
    GNUstep ProjectCenter - http://www.gnustep.org/experience/ProjectCenter.html
 
-   Copyright (C) 2004 Free Software Foundation
+   Copyright (C) 2004-2011 Free Software Foundation
 
    Authors: Serg Stoyan
+            Riccardo Mottola
 
    This file is part of GNUstep.
 
@@ -93,7 +94,7 @@
 {
   NSString  *path = nil;
 
-  ASSIGN(_fileTypes, fileTypes);
+  [super setAllowedFileTypes: fileTypes];
 
   path = [_browser path];
   [self validateVisibleColumns];
